platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/MapboxTest.java

```diff
@@ -85,8 +85,8 @@ public void testNoInstance() {
 
     expectedException.expect(MapboxConfigurationException.class);
     expectedException.expectMessage(
-      "\nUsing MapView requires calling Mapbox.getInstance(Context context, String apiKey) before "
-      + "inflating or creating the view."
+      "\nUsing MapView requires calling Mapbox.getInstance(Context context, String apiKey," +
+              " WellKnownTileServer wellKnownTileServer) before inflating or creating the view."
```

Comment:
CI complains that 

```
native/platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/MapboxTest.java:88:93: '+' should be on a new line. [OperatorWrap]

```

Reply:
Maybe it is easiest if you run locally `make android-check` to make sure that the linting is fine. Or maybe there are better local options, I don't know...

Reply:
Fixed that one. Hardcoded test